idl/.classpath

```diff
@@ -23,11 +23,6 @@
 			<attribute name="maven.pomderived" value="true"/>
 		</attributes>
 	</classpathentry>
-	<classpathentry excluding="**" kind="src" output="target/classes" path="target/generated-sources/idl">
```

Comment:
Please no unrelated changes.

Reply:
Couldn't change as the file was locked(read-only mode).